forth_src/platform.fs

```diff
@@ -0,0 +1,14 @@
+marker ---get-platform---
+
+\ use IRQ vector to identify which
+\ system we are running on
+: get-platform 
+$fffe @ case
+$ff48 of 64 endof
+$ff17 of 128 endof
+abort" cannot identify platform"
+endcase ;
```

Comment:
It seems wasteful to do this check in runtime, both in terms of memory and CPU cycles. Could it be better to introduce a constant that is set from assembly TARGET?